### DISCUSSION TITLE
Meeting Notes: Ultrade & ASA Stats about the possible ways for collaboration

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#313] [Meeting]
I had a Zoom meeting with Dan, [Ultrade](https://ultrade.org/) CEO today, initiated by Ultrade after TheShaman pointed to us.

They're going to offer custom/branded DEX and AMM solutions for third parties built on their infrastructure and we have talked about the possible ways for collaboration.

From our perspective, it has been emphasized that we're surely interested in resources that will help us research their platform. It is yet to be defined how data from their partners (using Ultrade platform under their websites under framed subdomain.ultrade.org or so) will be presented in ASA Stats, but we'd probably be able to distinct evaluated data sources and provide direct links to their partners instead of common ultrade.org links.

ASA Stats is surely interested in providing custom user widgets presenting their platform to the users who choose to select such a widget inside user settings.

I've explained that ASA Stats won't spend resources in the pre-DAO phase that aren't covered by our [roadmap](https://github.com/asastats/docs/blob/main/roadmap.md), therefore the implementation of their platform to build custom DEX/AMM inside asastats.com has to wait until DAO takes over and decide upon in an eventual voting session.

For future communication, a Discord group is created with Dan and Garrett from Ultrade, and ipaleka from ASA Stats. We'd be adding members to the group from our community when necessary.

**Comment by marucat27** [comment]
The thought of having such a feature like a dex/amm within Asa stats as a widget or other implementation is very exciting 👍
